libqtile/core/manager.py

```diff
@@ -360,6 +360,32 @@ def _process_screens(self) -> None:
             )
             self.screens.append(scr)
 
+    def reconfigure_screens(self, ev):
+        """
+        This configures screens upon firing of the screen_change hook.
```

Comment:
Maybe the comment should be changed or removed, since the connection between `reconfigure_screens` and the mentioned hook is now optional.

Reply:
Good call. Done.